### PR TITLE
New List: file_flavors_macros.txt

### PR DESCRIPTION
**Pull request by aidenmitchell**
TL;DR: Right now, some macro detections rely on a file extension in order to detect a document, which misses macros inside extension-less files. Using flavors to find documents is more reliable and ensures that files are not missed.

---
Rule example:

Existing rule:
```
type.inbound
and any(attachments, 
  .file_extension in~ $file_extensions_macros
  and file.oletools(.).indicators.vba_macros.risk == "high"
```

Fixed rule:
```
type.inbound
and any(attachments, 
  (.file_extension in~ $file_extensions_macros or .content_type in~ $file_flavors_macros)
  and file.oletools(.).indicators.vba_macros.risk == "high"
```

--- 
File example:

Malicious .xls file with macros, but no file extension (file name: `test`): 
image
https://analyzer.sublime.security?id=98d97d78-54f4-466d-a40c-1f1775fcf6d3

Same .xls file with macros, but _with_ file extension (file name: `test.xls`): 
image
https://analyzer.sublime.security?id=d4fe2018-8789-49e4-a24e-305699bbe193

### CODE REVIEW

**Comment by aidenmitchell** [comment]
The only issue here, is that the `content_type` surfaced in the MDM of the EML is not the same as the `flavors.mime` value, of which the latter is usually correct. 

See here (https://analyzer.sublime.security?id=d7952906-6fff-402d-9dd1-a57ae87ae91e):
image

image

**Comment by morriscode** [comment]
Is this still needed @aidenmitchell

**Comment by aidenmitchell** [comment]
I think we've corrected for this in other ways, so no, closing.
